ftest.sh

```diff
@@ -125,8 +125,8 @@ if ${TEARDOWN_ONLY:-false}; then
 fi
 
 # let's output to a dir in the tree
-rm -rf src/tests/ftest/avocado ./*_results.xml
-mkdir -p src/tests/ftest/avocado/job-results
+rm -rf install/lib/daos/TESTING/ftest/avocado ./*_results.xml
```

Comment:
It would appear that assigning `install/lib/daos/TESTING` (or whatever ends up being used) to a variable would make sense given its reuse.

Reply:
https://jira.hpdd.intel.com/browse/DAOS-3607
Will move CI test to DAOS RPM env.